climakitae/core.py

```diff
@@ -1,5 +1,8 @@
-from .selectors import DataSelector, _display_select, LocSelectorArea
+from .selectors import (DataSelector, _display_select, LocSelectorArea,
+                        UserFileChoices, _user_export_select, FileTypeSelector)
 from .data_loaders import _read_from_catalog
+from .data_export import _export_to_user
+from .metadata_update import transform_details
```

Comment:
is transform_details used in this file? not sure it needs to be imported in core.py